src/js/core/factories/GridColumn.js

```diff
@@ -81,6 +81,23 @@ angular.module('ui.grid')
    */
 
   /**
+   * @ngdoc property
```

Comment:
It looks like you added the documentation twice here. Please remove the one without the example.